utils/cups-browsed.c

```diff
@@ -430,6 +430,8 @@ static unsigned int BrowseInterval = 60;
 static unsigned int BrowseTimeout = 300;
 static uint16_t BrowsePort = 631;
 static browsepoll_t **BrowsePoll = NULL;
+int NewBrowsePollQueuesShared = 0;
```

Comment:
@V1krant, could you add the "static" here? Thanks.

Reply:
Yeah sure.